cdap-hbase-compat-base/src/main/java/co/cask/cdap/replication/ReplicationStatusKey.java

```diff
@@ -26,18 +26,28 @@
  */
 public final class ReplicationStatusKey {
 
+  private final String rowType;
+  private final String regionName;
+  private final String rsID;
   private final byte[] key;
 
   public ReplicationStatusKey(byte[] rowType, byte[] regionName, byte[] regionServerID) {
```

Comment:
In previous commits, I mentioned that row type and regionserver id are constant and should be converted only once. This is still true, but lets do it in a separate commit to limit the changes we're making in this PR. 

The way you would do that is to create a factory class that takes rowtype and regionserverid in its constructor, and then has a method to create these keys that takes the regionName.